src/Model/TransformerInterface.php

```diff
@@ -17,6 +17,48 @@
 
 /**
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * @phpstan-type BlockContent array{
+ *   id: int|string|null,
+ *   name?: string|null,
+ *   enabled: boolean,
+ *   position: int|null,
+ *   settings: array<string, mixed>,
+ *   type: string,
+ *   created_at: int|string|null,
```

Comment:
This was copied from a previous version of the 4.x branch, can you update it?

Reply:
what exactly need to be updated?
or should i just leave it out for now? (it does exist in 4.x anyway)

Reply:
Check for differences on 4.x, multiple things changed.

Reply:
i compared, besides the order i don't see any difference

Reply:
Different order mean conflicts during the merge :/

Might indeed be better to not use type in 3.x if it's not useful to have less conflict during the merge. 🐙

Reply:
> i compared, besides the order i don't see any difference
> 
> again, should i just remove this and let it only be for 4.x?

There is at least one difference. On 4.x created_type is numeric-string|null instead of int|string|null. And there are probably more, like type is string|null...